### PR TITLE
WIP: Feat/video inline

### PR DESCRIPTION
**Pull request by nehasri89**
This adds an inline video in article

### CODE REVIEW

File: packages/article/package.json [hunk 1]
```diff
@@ -80,7 +80,12 @@
     "@times-components/tracking": "0.10.5",
     "@times-components/video-label": "0.1.21",
     "lodash.get": "4.4.2",
-    "prop-types": "15.6.0"
+    "prop-types": "15.6.0",
+    "react-apollo": "2.1.0-rc.3",
+    "apollo-client": "2.2.3",
+    "apollo-link-http": "1.5.2",
+    "apollo-cache-inmemory": "1.1.5",
+    "@storybook/addon-knobs": "3.3.15"
```
Comment (fourlastor) [inline packages/article/package.json]: this should go in dev dependencies

File: packages/article/package.json [hunk 2]
```diff
@@ -80,7 +80,12 @@
     "@times-components/tracking": "0.10.5",
     "@times-components/video-label": "0.1.21",
     "lodash.get": "4.4.2",
-    "prop-types": "15.6.0"
+    "prop-types": "15.6.0",
+    "react-apollo": "2.1.0-rc.3",
+    "apollo-client": "2.2.3",
+    "apollo-link-http": "1.5.2",
+    "apollo-cache-inmemory": "1.1.5",
```
Comment (fourlastor) [inline packages/article/package.json]: why do we need all these deps here? they should be in provider

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/e4b25641ca2cc66d634295b3e5e20e5dd47a2e3e/index.html

**Comment by nehasri89** [comment]
Closing this so and opening another one https://github.com/newsuk/times-components/pull/824

**Pull request closed without merging**